velox/expression/tests/CastExprTest.cpp

```diff
@@ -1076,6 +1067,13 @@ TEST_F(CastExprTest, primitiveValidCornerCases) {
     testCast<std::string, bool>("boolean", {"0"}, {false});
     testCast<std::string, bool>("boolean", {"t"}, {true});
     testCast<std::string, bool>("boolean", {"true"}, {true});
+    testCast<std::string, bool>("boolean", {"false"}, {false});
```

Comment:
Could you also add a test case of casting 'f' to boolean? Also for lower/upper cases and a mix of them.